community/girara/depends

```diff
@@ -0,0 +1,5 @@
+glib
+gtk+3
+json-c
+meson make
+ninja make
```

Comment:
This line can now be removed due to my changes here: https://github.com/kisslinux/repo/commit/e27b638dda62dd9f090c978507d01fa5237e5a45

Reply:
Sorry, it's not clear for me. As I understand your change, I can change

`+ninja make`

to 

`+samurai make`

The other lines will remains unchanged. Is it right ?

Reply:
Just remove it entirely. `meson` will pull it in automatically. :)

Reply:
Ok, I'll remove this line, but I still don't understand, sorry..... 
My reasoning is the following: if samurai is not in the dependency, the package could be theoretically built without samurai. I understand meson will automatically use samurai instead of ninja. Ok. But if samurai is not installed there is a problem for me. 

Unless samurai is installed in the core system. Could you bring me the light ? :sweat:

Reply:
The `meson` package depends on `samurai`.

Reply:
Ok!!! Thank you! Next time I'll check the dependencies.....